client/src/pages/UserAvatar.jsx

```diff
@@ -0,0 +1,34 @@
+import { HiUserCircle, HiPencil } from 'react-icons/hi';
+
+const UserAvatar = (prop) => {
```

Comment:
please spread the props like other components, then use proptypes. you can see what i mean by looking at other components and this should be in the component folder but the pages' folder because it's a component not a page.

Reply:
Done!